### ISSUE TITLE
[BUG] Custom classnames broke with npm / node minor release

### ISSUE THREAD

**Issue by hsjobeki** [francoismassart/eslint-plugin-tailwindcss#386]
**Describe the bug**
Eslint reports invalid linting errors. The reported classes do exist and they properly work with one minor version less.

```bash
   9:10  error  Classname 'hero' is not a Tailwind CSS class!          tailwindcss/no-custom-classname
  10:12  error  Classname 'hero-content' is not a Tailwind CSS class!  tailwindcss/no-custom-classname
  20:15  error  Classname '!link' is not a Tailwind CSS class!         tailwindcss/no-custom-classname
```

**To Reproduce**
1. Install and setup tailwind with some thrid party extension library like dasyui
2. use some of the classes provided by the plugin
2. run eslint with the rule "no-custom-classname"

**Environment (please complete the following information):**

- Works fine with:
  - nodejs: v22.11.0
  - npm: 10.9.0
- Breaks with:
  - nodejs: v22.12.0 
  - npm: 10.9.0

**Additional context**
All dependency versions and their hashes didn't change at all.

 "node_modules/eslint-plugin-tailwindcss":  "3.18.0",
 "node_modules/eslint": "version": "8.57.0",
 "node_modules/tailwindcss": "3.4.3"

**Comment by Codex-** [comment]
I encountered this as an issue when updating to the latest vscode, so for anyone that runs into this, you need to downgrade in the meantime to 1.99: https://code.visualstudio.com/updates/v1_99

**Comment by gnpwaritschlager** [comment]
Fyi this can also happen if your tailwindcss config file path isn't configured correctly in eslint config file. This leads to TW plugin class names not being picked up by this project.
